src/modules/universal-discovery/components/search/search.component.js

```diff
@@ -86,6 +88,33 @@ export default class SearchComponent extends Component {
         }
     }
 
+    /**
+     * Renders the language filter
+     *
+     * @method renderLanguageFilter
+     */
+    renderLanguageFilter() {
+        const selectAttrs = {
+            className: 'form-control',
+            ref: this.setSearchLanguageRef
+        };
+
+        const { languages } = this.props;
```

Comment:
Please move this to the top of the method. Destructs of `state` or `props` should be on the top.

Reply:
You should also add `languages` to the `propTypes`